src/crimson/os/alienstore/alien_store.cc

```diff
@@ -66,6 +66,16 @@ AlienStore::AlienStore(const std::string& path, const ConfigValues& values)
   const auto num_threads =
     cct->_conf.get_val<uint64_t>("crimson_alien_op_num_threads");
   std::vector<uint64_t> cpu_cores = _parse_cpu_cores();
+
+  if (cpu_cores.empty()) {
+    if (long nr_cpus = sysconf(_SC_NPROCESSORS_ONLN); nr_cpus != -1) {
+      for (int i = 1; i <= ((nr_cpus > 3) ? (nr_cpus - 3) : 0); i++) {
+        cpu_cores.push_back(nr_cpus - i);
```

Comment:
might want to 

1. define a `static constexpr int N_CORES_FOR_ALIENSTORE = 4`.
2. add a comment noting that we are trying to use the last 4 cores for alienstore.
3. put something like
```c++
// reserve a single core for reactor
if (cpu_cores.empty() && nr_cpus > N_CORES_FOR_ALIENSTORE) {
  for (uint64_t core = nr_cpus - N_CORES_FOR_ALIENSTORE; core < nr_cpus; core++) {
    cpu_cores.push_back(core);
  }
}
```

Reply:
Done:-)